Routes/api-router.js

```diff
@@ -1,5 +1,7 @@
 const apiRouter = require("express").Router();
 
-apiRouter.route("/").get(sendApiEndpoints);
+apiRouter.get("/", () => {
+  res.sendStatus(200);
```

Comment:
When doing just one thing, you can  simply do this:

`apiRouter.get("/", () => res.sendStatus(200));`

Reply:
I'm confused sorry, I removed .get(sendApiEndpoints) and had res.sendStatus(200) in the most recent pull?

Reply:
```
apiRouter.get("/", () => {
  res.sendStatus(200);
});
```
Can be written as:

```apiRouter.get("/", () => res.sendStatus(200));```